src/components/views/rooms/AppsDrawer.js

```diff
@@ -131,6 +132,9 @@ module.exports = React.createClass({
             '$matrix_room_id': this.props.room.roomId,
             '$matrix_display_name': user ? user.displayName : this.props.userId,
             '$matrix_avatar_url': user ? MatrixClientPeg.get().mxcUrlToHttp(user.avatarUrl) : '',
+
+            // Namespaced for Riot
+            '$riot_theme': SettingsStore.getValue("theme"),
```

Comment:
Can we please change '$riot_theme' to '$theme' -- See main thread comment.

Reply:
The solution of namespacing themes makes a lot more sense. Should the variable instead be `$matrix_theme` or just change it to `$theme`?

Reply:
After thinking about it for a bit, I decided not to because it's not related to matrix - it's related to the client only.

@rxl881 updated.

Reply:
Great. Thanks!